heft-plugins/heft-jest-plugin/package.json

```diff
@@ -15,7 +15,8 @@
     "build": "heft build --clean",
     "start": "heft test --clean --watch",
     "_phase:build": "heft build --clean",
-    "_phase:test": "heft test --no-build"
+    "_phase:test": "heft test --no-build",
+    "postinstall": "node ./src/patches/jestScriptTransformerPatch.js"
```

Comment:
Does the `src` folder ship? If not, this won't work when the package is published. Likely will need to put it in a folder that exists at both workspace install time and shipped install time, or else manually add `src/patches/jestScriptTransformerPatch.js` to the package manifest

Reply:
Yeah we don't publish the `src` folder. Moved it to `includes` which does get published.